pkg/sql/show_tenant.go

```diff
@@ -242,7 +244,12 @@ func (n *showTenantNode) Values() tree.Datums {
 				// microsecond. In that case a user may want to cutover to a rounded-up
 				// time, which is a time that we may never replicate to. Instead, we show
 				// a time that we know we replicated to.
-				replicatedTimestamp, _ = tree.MakeDTimestampTZ(minIngested.GoTime().Truncate(time.Microsecond), time.Nanosecond)
+				minIngestedMicro := minIngested.GoTime().Truncate(time.Microsecond)
+				replicatedTimestamp, _ = tree.MakeDTimestampTZ(minIngestedMicro, time.Nanosecond)
+
+				nowMicro := timeutil.Now().Truncate(time.Microsecond)
```

Comment:
is p.EvalContext().GetStmtTimestamp more correct than observing current wall time?

Reply:
good point. done. note that I had to add another field to `showTenantNode`